cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -52,9 +68,18 @@ describe('Pods', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, ()
     });
 
     it('pagination is visible and user is able to navigate through pods data', () => {
+      cy.tableRowsPerPage(10);
+      cy.reload(true);
+      // The pods count returned just before the assertions are out of sync with the created pods during the setup
+      // Waiting seems to mitigate this sync issue. This behavior seems consistent/reproducible on HA Rancher
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(30000);
+
       // get pods count
       cy.getRancherResource('v1', 'pods').then((resp: Cypress.Response<any>) => {
-        const count = resp.body.count;
+        // only retrieve the test data objects
+        const rExp = /(e2e-test-\d{13}-ns\d|default)/;
```

Comment:
Something doesn't quite seem right
- Pods are created in nsName1 and nsName2 namespaces
- All pods are fetched and then filtered on `/(e2e-test-\d{13}-ns\d|default)/` (though could just match nsName1 & nsName2)
- Filtered pods count is checked against the lists pagination total results count

However the ns filter is set to none, which would affect the resource count in the list. In theory it should include all the system pods.

Reply:
I found out during the test suite execution that there are some other resources created that landed in the `default` namespace. The tests are filtering using the `Only User Namespaces` which included `default`'s resources.

Isn't that expected? I can open an issue if not.

Reply:
I don't think they're filtering on anything. The line `cy.updateNamespaceFilter('local', 'none', '{\"local\":[]}');` should mean `All Namespaces` is selected. `{\"local\":[\"all://user\"]}` would be the value to show `Only User Namespaces` (which is set in the `after`).  This can be validated by either navigating to the cluster or making an api call to <rancher url>/v1/userpreferences and looking at the `ns-by-clusters` value.

Reply:
Well, I'll need to double check if that setup configuration is getting properly applied in the test as I observed the assertions being made on the `Only User Namespaces` filter not on `All Namespaces`

Reply:
I had to make the test update both the namespace filter feature and the rows per page on the same `userpreferences` `PUT` There was a conflict when I only updated the rows per page the Namespace filter is updated to the default  `all://user`